timescaledb/getting-started/add-data.md

```diff
@@ -1,53 +1,64 @@
-* temp_min_c: Minimum temperature
-* temp_max_c Maximum temperature
-* pressure_hpa: Atmospheric pressure (at sea level) in hPa
-* humidity_percent:  Humidity as a percentage of maximum water vapor
-* wind_speed_ms: Wind speed in meters per second
-* wind_deg: Wind direction, degrees (meteorological)
-* rain_1h: Rain volume for the last hour, mm
-* rain_3h: Rain volume for the last 3 hours, mm
-* snow_1h: Snow volume for the last hour, in mm
-* snow_3h: Snow volume for the last 3 hours, in mm
-* clouds_percent: Cloudiness as a percentage
-* weather_id: Weather condition id
-```
-
-## Accessing the dataset
-
-We provide a CSV file with commands for inserting the data into your TimescaleDB instance.
-
-Download the CSV file (in ZIP format) below and insert it into your database from psql.
-
-Download CSV: <tag type="download">[weather_data.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/weather_data.zip)</tag>
-
-After unzipping the file, use the following command (which assumes `weather_data.csv` is located in your current directory):
-
-```sql
--- copy data from weather_data.csv into weather_metrics
-\copy weather_metrics (time, timezone_shift, city_name, temp_c, feels_like_c, temp_min_c, temp_max_c, pressure_hpa, humidity_percent, wind_speed_ms, wind_deg, rain_1h_mm, rain_3h_mm, snow_1h_mm, snow_3h_mm, clouds_percent, weather_type_id) from './weather_data.csv' CSV HEADER;
-```
-Now that you’re up and running with historical data inside TimescaleDB and a
-method to ingest the latest data into your database, let’s start querying the data.
+# Add time-series data
+
+To help get you familiar with TimescaleDB's features, you’ll need some
+sample data to play around with. For this tutorial, we provide real-time financial 
+stock data provided by ['Twelve Data'][twelve-data]. The data set contains time-series 
+stock prices from the last three months. 
+
+<highlight type="note">
+For information on how to get more from 'Twelve data', check out our [tutorial][script-twelve-data] on how to 
+ingest stock data into TimescaleDB.
+</highlight>
+
+### About the dataset
+
+The dataset contains tick level (second by second) stock data along with further information on company symbols. 
+There are two tables that you will ingest, a hypertable `stocks_real_time` and a regular table `company`. 
+
+Details on the data set are as follows:
+`stocks_real_time`: Stock data indicating quotes for stock prices for a given second during trading hours
+    * time: timestamp column incrementing at a tick level (second by second)
+    * symbol: symbols representing a company, mappings found in the `company` table
+    * price: stock quote price for a company at the given timestamp
+    * day_volume: how many shares are traded each day, NULL values indicate the market is closed
+
+`company`: mapping for symbols to company names
+    * symbol: the symbol representing a company name
+    * name: corresponding company name
```

Comment:
```suggestion
Details on the data set are as follows:

`stocks_real_time`: Stock data indicating quotes for stock prices for a given second during trading hours

```bash
    * time: timestamp column incrementing at a tick level (second by second)
    * symbol: symbols representing a company, mappings found in the `company` table
    * price: stock quote price for a company at the given timestamp
    * day_volume: how many shares are traded each day, NULL values indicate an after hours trade
```

`company`: mapping for symbols to company names

```bash
    * symbol: the symbol representing a company name
    * name: corresponding company name
```
```

Look at the formatting in the preview for this one. I was surprised to see some bash formatting cells (which is totally fine) but if that's what we want I think it probably needs to be specific to be safe.